src/components/TextInput.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import { 
+    View, 
+    Text,
+    Image, 
```

Comment:
The same about indentation and ESLint, comments about that that I left in the MainStackNavigator.js file and about the style files.